cmd/kusk/cmd/mock.go

```diff
@@ -456,7 +460,11 @@ func decorateLogEntry(entry mockingServer.AccessLogEntry) string {
 func init() {
 	rootCmd.AddCommand(mockCmd)
 	mockCmd.Flags().StringVarP(&apiSpecPath, "in", "i", "", "path to openapi spec you wish to mock")
-	mockCmd.MarkFlagRequired("in")
+	if err := mockCmd.MarkFlagRequired("in"); err != nil {
```

Comment:
I dont think we need to check the error here. We have the flag defined on the line above so we know that `MarkFlagRequired` will return a nil error